### ISSUE TITLE
error: invalid conversion from ‘int’ to ‘const char*’

### ISSUE THREAD

**Issue by rounaksingh** [SergiusTheBest/plog#176]
Hi,
I am using plog in linux platform; g++ compile is old -- Version 4.4.7. It only supports std=c++0x.

While the operator<< is working for string or const char* when I am compiling Hello and ColorConsole in samples folder of the repo, however when I am trying to compile SetFileName or Demo example. I am getting following error due to conversion from int to const char. Also, this error is persistant with long, short like data types.

```
[me@something SetFileName ]$ g++ -Wall -std=c++0x -I../../include Main.cpp -o main
In file included from ../../include/plog/Appenders/IAppender.h:2,
                 from ../../include/plog/Logger.h:2,
                 from ../../include/plog/Log.h:7,
                 from Main.cpp:5:
../../include/plog/Record.h: In function ‘void plog::detail::operator<<(plog::util::nostringstream&, const T&) [with T = int]’:
../../include/plog/Record.h:199:   instantiated from ‘plog::Record& plog::Record::operator<<(const T&) [with T = int]’
Main.cpp:17:   instantiated from here
../../include/plog/Record.h:61: error: invalid conversion from ‘int’ to ‘const char*’
../../include/plog/Record.h:61: error:   initializing argument 1 of ‘std::basic_string<_CharT, _Traits, _Alloc>::basic_string(const _CharT*, const _Alloc&) [with _CharT = char, _Traits = std::char_traits<char>, _Alloc = std::allocator<char>]’
```
 
Am I doing something wrong here? or this is won't work for my compiler.
Thank you.

**Comment by SergiusTheBest** [comment]
Hi,

Let me check that.

**Comment by SergiusTheBest** [comment]
I fixed building on GCC 4.4-4.7. Please, try it.

**Comment by rounaksingh** [comment]
Hi @SergiusTheBest ,
I tried your fix. It is working.

Appreciate it!
-Ro
